internal/service/project/data_source_projects.go

```diff
@@ -134,6 +134,34 @@ func (d *ProjectsDS) Schema(ctx context.Context, req datasource.SchemaRequest, r
 								},
 							},
 						},
+						"ip_addresses": schema.SingleNestedAttribute{
```

Comment:
can some part of the schema be reused from the RS or it's hard because here all are Computed?

Reply:
The main limitation is that they are different schema types, `terraform-plugin-framework/datasource/schema` and `terraform-plugin-framework/resource/schema` respectively.